<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DisplayWidget" name="displayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>256</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tilesComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>501</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="TilesWidget" name="tilesWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="CharacterButton" name="characterButtonBear">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>96</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/characters/ch_bear_32x24.png</normaloff>:/characters/ch_bear_32x24.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>64</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CharacterButton" name="characterButtonDog">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>96</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/characters/ch_dog_16x10.png</normaloff>:/characters/ch_dog_16x10.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>64</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CharacterButton" name="characterButtonSpiderSmall">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>96</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/characters/ch_spider_16x8.png</normaloff>:/characters/ch_spider_16x8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>64</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CharacterButton" name="characterButtonSpiderBig">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>96</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/characters/ch_spider_24x16.png</normaloff>:/characters/ch_spider_24x16.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>64</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CharacterButton" name="characterButtonPlayer">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>96</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/characters/ch_player_8x8.png</normaloff>:/characters/ch_player_8x8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>64</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CharacterListView" name="characterListView">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enemyDeleteButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Enemy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="leftRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rightRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="ParamCheckBox" name="showCollidersCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show colliders</string>
    </property>
   </widget>
   <widget class="ParamCheckBox" name="showWalkZonesCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show walk zones</string>
    </property>
   </widget>
   <widget class="ParamCheckBox" name="showAIVisionCheckBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show AI vision</string>
    </property>
   </widget>
   <widget class="ParamCheckBox" name="showAITargetCheckBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show AI target</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerHealthEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHealth">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInitialBehavior">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial behavior:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInitialBehavior">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPatrolBehavior">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Patrolling behavior:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPatrollingBehavior">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInitialDirection">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial direction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEnemies">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enemies:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerAmmoEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAmmo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ammo:</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGrid"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep"/>
   <addaction name="actionEdit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuLevel">
    <property name="title">
     <string>Level</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionImportTileset"/>
    <addaction name="actionResize"/>
    <addaction name="separator"/>
    <addaction name="actionImportImage"/>
    <addaction name="actionExportImage"/>
    <addaction name="separator"/>
    <addaction name="actionSnapshot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLevel"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>New level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/images.qrc">
     <normaloff>:/icon_play.png</normaloff>:/icon_play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play emulation</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/images.qrc">
     <normaloff>:/icon_pause.png</normaloff>:/icon_pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause emulation</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="Images/images.qrc">
     <normaloff>:/icon_step.png</normaloff>:/icon_step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Step emulation</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/images.qrc">
     <normaloff>:/icon_grid.png</normaloff>:/icon_grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Toggle grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/images.qrc">
     <normaloff>:/icon_edit.png</normaloff>:/icon_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Toggle Edit Mode</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size...</string>
   </property>
  </action>
  <action name="actionOffset">
   <property name="text">
    <string>Offset...</string>
   </property>
  </action>
  <action name="actionFrom_png">
   <property name="text">
    <string>From png...</string>
   </property>
  </action>
  <action name="actionImportImage">
   <property name="text">
    <string>Import image...</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="text">
    <string>Export image...</string>
   </property>
  </action>
  <action name="actionImportTileset">
   <property name="text">
    <string>Import tileset...</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Meta+R</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize...</string>
   </property>
   <property name="shortcut">
    <string>Meta+Alt+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header>displaywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TilesWidget</class>
   <extends>QWidget</extends>
   <header>TilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterButton</class>
   <extends>QToolButton</extends>
   <header>CharacterButton.h</header>
  </customwidget>
  <customwidget>
   <class>CharacterListView</class>
   <extends>QListView</extends>
   <header>CharacterListView.h</header>
  </customwidget>
  <customwidget>
   <class>ParamCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ParamCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
